src/commands/transaction/reconstruct_transaction/mod.rs

```diff
@@ -240,6 +240,12 @@ fn action_transformation(
         Action::Delegate(_) => {
             panic!("Internal error: Delegate action should have been handled before calling action_transformation.");
         }
+        Action::DeployGlobalContract(_) => {
+            todo!("The feature is not implemented yet.");
+        }
+        Action::UseGlobalContract(_) => {
+            todo!("The feature is not implemented yet.");
+        }
```

Comment:
nit: text of panic may be changed to mention `Action::DeployGlobalContract` + `Action::UseGlobalContract`

+issue may be created to track these `unimplemented!()` panics formally, if merging full support for these isn't expected today